identify/identify.py

```diff
@@ -159,6 +159,15 @@ def parse_shebang(bytesio):
     cmd = tuple(_shebang_split(first_line.strip()))
     if cmd and cmd[0] == '/usr/bin/env':
         cmd = cmd[1:]
+    if cmd and cmd[0].endswith('nix-shell'):
```

Comment:
should this be `if cmd == ('nix-shell',):`

Reply:
Thanks for this catch! Solved in https://github.com/chriskuehl/identify/pull/130/commits/00f56a68ea5d6ace7ee362ea5f3bd918ce2b204a